css/Helpers/_christmas.scss

```diff
@@ -0,0 +1,15 @@
+$color-gray: #951f2b;
```

Comment:
katalog /helpers to nie najlepsze miejsce na theme, wrzuc go po prostu do katalogu /css zeby byl na tym samym poziomie co glowny plik style.css, ewentualnie dodaj nowy katalog /themes